backend/core/src/listings/listings.service.ts

```diff
@@ -155,11 +155,16 @@ export class ListingsService {
     if (!listing) {
       throw new NotFoundException()
     }
+    let availableUnits = 0
     listingDto.units.forEach((unit) => {
       if (!unit.id) {
         delete unit.id
       }
+      if (unit.status === "available") {
+        availableUnits++
+      }
     })
+    listingDto.unitsAvailable = availableUnits
```

Comment:
Do you know why this isn't a computed field? Seems like maybe it should be? 🤷

Reply:
yeah, I'm not really sure either, I can ask @seanmalbert about it during stand up to see what he thinks

Reply:
@emilyjablonski and @YazeedLoonat, we can move it to a computed field if you think that makes more sense, though I'd rather do the work once on save, instead of on each request.